### ISSUE TITLE
Nightly GitHub Action failure

### ISSUE THREAD

**Issue by robert-scheck** [rpki-client/rpki-client-portable#65]
Our nightly GitHub Action for the container image [fails](https://github.com/rpki-client/rpki-client-container/runs/6123020393?check_suite_focus=true#step:9:623) since yesterday with:

```
#34 16.10 Copying rrdp_notification.c
#34 16.11 Copying rrdp_snapshot.c
#34 16.11 Copying rrdp_util.c
#34 16.12 Copying rsync.c
#34 16.12 Copying tal.c
#34 16.12 Copying validate.c
#34 16.12 Copying version.h
#34 16.13 Copying x509.c
#34 16.14 Applying patch patches/0001-Allow-overriding-default-user-and-file-locations.patch
#34 16.15 1 out of 6 hunks FAILED -- saving rejects to file main.c.rej
```

**Comment by cjeker** [comment]
Should be fixed in HEAD aka a9253f3a738989a5b42335d4597cc033ecbea633

**Comment by robert-scheck** [comment]
Confirming [successful run](https://github.com/rpki-client/rpki-client-container/actions/runs/2205682084) with commit a9253f3a738989a5b42335d4597cc033ecbea633 - thank you.
